Makefile

```diff
@@ -151,5 +151,8 @@ build-image-tnf:
 		-t ${REGISTRY}/${TNF_IMAGE_NAME}:${TNF_VERSION} \
 		-f Dockerfile .
 
+# Generates the classification.js file and creates anew result-embed.html
 classification-js:
 	./tnf generate catalog javascript > script/classification.js
+	sed '/<script src=".\/classification.js"><\/script>/e echo "<script>";cat  script/classification.js; echo "<\/script>"' script/results.html > script/results-embed.html
```

Comment:
Put every command on a new line, `;` is not needed.

Reply:
the ";" is used in this case because the 3 commands are executed by sed

Reply:
You're right. For better formatting, add space after `;`, and remove space after `cat`.

Reply:
fixed